src/data/projects/newrelic-newrelic-quickstarts.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "newrelic-quickstarts",
+  "fullName": "newrelic/newrelic-quickstarts",
+  "slug": "newrelic-newrelic-quickstarts",
+  "owner": {
+    "login": "newrelic",
+    "type": "Organization"
+  },
+  "title": "New Relic quickstarts",
+  "supportUrl": null,
+  "githubUrl": "https://github.com/newrelic/newrelic-quickstarts",
+  "permalink": "https://opensource.newrelic.com/projects/newrelic/newrelic-quickstarts",
+  "defaultBranch": "main",
+  "contributingGuideUrl": "https://github.com/newrelic/opensource-website/blob/develop/CONTRIBUTING.md",
+  "iconUrl": null,
+  "shortDescription": "Observability building blocks like instrumentation, dashboards and alerts",
+  "description": "New Relic One quickstarts help accelerate your New Relic journey by providing immediate value for your specific use cases",
+  "ossCategory": "community-plus",
```

Comment:
I think we want this to be `community` since the `newrelic-quickstarts` repository is still running that banner. @jpvajda Do you know if that's changing?

Reply:
oh yes i think it should reflect what's in repo currently, we can always change this if that changes going forward

Reply:
the repo for quickstarts is fully supported by us and virtuoso, it's definitely community-plus.

https://opensource.newrelic.com/oss-category/#community-plus